server/errors.ts

```diff
@@ -208,3 +208,19 @@ export function ClientClosedRequestError(
     id: "client_closed_request",
   });
 }
+
+export function DiscordGuildError(
+  message = "User is not a member of the required server"
```

Comment:
nit: Can the message on this be clarified to specify that it refers to discord servers?

Reply:
Changed to `User is not a member of the required Discord server`